todo_app/templates/index.html

```diff
@@ -7,11 +7,22 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+
+  <form action="/items" method="post">
+    <div class="form-group">
+        <label>New item:</label>
+        <input type="text" class="form-control" name="item-title">
+    </div>
+      <button type="submit" class="btn btn-primary">Post</button>
+  </form>
+
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+          {% for item in items %}
+              <li class="list-group-item">{{ item.title }}<button type="button" >{{ item.status }}</button></li>
```

Comment:
Why have you used a button here? Clicking the status performs no action.

I think you should use a `<span>` tag instead.